code_curation2_submissions/Santiage_rs_seed_repo_generator.m

```diff
@@ -1,54 +1,36 @@
 function [Result] = rs_seed_repo_generator(Seeds,BOG_rsMRI,PSN_rsMRI,MainDataFolder,PipelineName)
 % function [ADataDir,AMaskFilename, AROIDef,AResultFilename, ACovariablesDef, VorR_tag, MainOutputFolder] = seed_repo_generator(Seeds,BOG_rsMRI,PSN_rsMRI,MainDataFolder,PipelineName)
```

Comment:
The first line of the header text should be exact the same as the function declaration.